app/helpers/orders_helper.rb

```diff
@@ -25,4 +25,58 @@ def cancel_order(order)
     render "orders/cancel_order_button", order: order unless
     order.status == "Delivered"
   end
+
+  def all_orders_count
+    current_shop_owner.shop.orders.count
+  end
+
+  def page_links
+    content_tag :div, style: "display: inline" do
+      order_count = total_pages
```

Comment:
@andela-anwocha you can remove this extra variable and use `total_pages` instead